app/TabNavigation.tsx

```diff
@@ -16,8 +16,16 @@ export function TabNavigation(): ReactElement {
 
   return (
     <Wrapper>
-      <div style={{ display: "flex" }}>
+      <div
+        style={{
+          display: "flex",
+          backgroundColor: "black",
```

Comment:
Any reason not to use a colour from your colour library

Reply:
yeah poop https://github.com/auberginearms/groceryDatabaseApp/pull/18/commits/346cad905ba15589c328b42b51f6709d5d04e9a8